### PR TITLE
Add dev log for 24.01

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: DEVELOPMENTLOG.md [hunk 1]
```diff
@@ -1,5 +1,75 @@
...
+Lastly, we've changed the way the land style 'paint' tool works. The land style selector is
+now hidden until the paint tool is activated. In addition, the tool no longer confusingly
+inherits corner selection from the general land tool. We hope this leads to more intuitive
+use of the land style paint tool.
+
+## Make viewport see-through flags more granular (#2231)
+
+The way hiding viewport elements works in Locomotion has often been suggested for improvement.
+Our codebase is now in such a state where we could finally rework this. The see-through flags
+are now no longer group several kinds of tile elements together, enabling players to control them
```
Comment (LeftofZen) [inline DEVELOPMENTLOG.md:34]: remove `are now`

Comment (AaronVanGeffen) [reply DEVELOPMENTLOG.md:34]: Oh, that's a typo alright.
```suggestion
now no longer group several kinds of tile elements together, enabling players to control them
```

Comment (LeftofZen) [reply DEVELOPMENTLOG.md]: or `The see-through flags are now more granular and no longer group several...

File: DEVELOPMENTLOG.md [hunk 2]
```diff
@@ -1,5 +1,75 @@
...
+to the windows wasn't the prettiest, however. This month, we completely refactored this,
+turning the respective `handleInput` events into proper `keyUp` window events. This makes
+things much more maintainable, and indeed easier to add such events in the future.
+
+## Refactor and split up the Input namespace (#2246, #2247)
+
+The changes to key input handling discussed in the previous section cleaned up the Input
+namespace quite a bit. However, it also revealed plenty more work to be done, separating core
+input logic from gameplay functions. We started by moving screenshot trigger logic out of the
+Input namespace,and moved a bunch of viewport interaction code to its own code unit as well.
```
Comment (LeftofZen) [inline DEVELOPMENTLOG.md:70]: missing a space after the comma

File: DEVELOPMENTLOG.md (hunk 1 above)
Comment (AaronVanGeffen) [reply DEVELOPMENTLOG.md]: Better, yes :+1: I'll amend.

**Review by LeftofZen — APPROVED** [review]

**Pull request merged**